.github/workflows/slack-notification.yml

```
@@ -0,0 +1,16 @@
+name: PR Slack Notification
+on: [pull_request, pull_request_review]
+
+jobs:
+  create-pull-request:
+    runs-on: ubuntu-latest
+    name: Run
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Fire Notification
+        uses: Lubycon/github-reviewer-slack-noti-action@v1.2.0
```

```suggestion
        uses: Lubycon/github-reviewer-slack-noti-action@v1.2.1
```

무섭님 `1.2.0` 버전은 버그가 있어용! `1.2.1` 버전으로 올려주시면 감사하겠습니다! 🙏

---

처음이라 잘 몰랐는데 알려주셔서 감사합니다! 🙏

---

몰랐는데 바로 알려주셔서 감사합니다 동욱님! 🙏